### PR TITLE
Create PR_template.md

### PR DESCRIPTION
**Pull request by kichloo**
Added the PR_template

### CODE REVIEW

**Comment by kichloo** [comment]
Fixes #32

**Comment by kichloo** [comment]
Fixes #32

**Comment by kichloo** [comment]
@rajats98 Pl review the PR.

**Review by rajats98 — CHANGES_REQUESTED** [review]
Hey
Check this once: 
https://github.com/CropAi/Data-Modeling/blob/master/.github/ISSUE_TEMPLATE/pull_request_template.md
Use similar directory structure for your submission,
Remove the testing part
Add all the check boxes given on above link

**Comment by kichloo** [comment]
@rajats98 Done!

File: PR_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Fixes issue
+Closes: #issue_number
+
```
Comment (rajats98) [inline PR_template.md:4]: addresses: #issue_number (if multiple persons are assigned, the issue)
Add this line!

**Review by rajats98 — CHANGES_REQUESTED** [review]
Move file to directory similar to the file(whose link was shared, before) 
i.e.  .github/TEMPLATES/pull_request_template.md
or .github/ISSUE_TEMPLATE/pull_request_template.md

**Pull request merged**